### PR TITLE
Ensure list type increases correctly 

### PR DESCRIPTION
**Pull request by juliaroldi**
When commit a list chain, verify if the next element index is the same number of the start number of the next list. If verification returns true, increases the start number of the next list. Also use the start attribute to set numbering list style properly. 

testListIncresed

testSetNumbering

### CODE REVIEW

**Comment by JiuqingSong** [comment]
I'm curious when was this regressed?

**Comment by juliaroldi** [comment]
> I'm curious when was this regressed?

This code is being there for at least 6 month according to git history. Looking the unit tests, it was made to cause this behavior.

**Comment by juliaroldi** [comment]
I merged both codes, now both scenarios are fixed. 
listIncrease

File: package.json [hunk 1]
```diff
@@ -77,5 +77,8 @@
         "hooks": {
             "pre-commit": "pretty-quick --staged"
         }
+    },
+    "dependencies": {
+        "ori": "^0.1.9"
```
Comment (JiuqingSong) [inline package.json:82]: is this change intentional?

File: packages/roosterjs-editor-dom/lib/list/VListChain.ts [hunk 2]
```diff
@@ -110,7 +110,13 @@ export default class VListChain {
 
         for (let i = 0; i < lists.length; i++) {
             const list = lists[i];
-            list.start = list.start > 1 ? list.start : lastNumber + 1;
+
+            //If there is a list chain sequence, ensure the list chain keep increasing correctly
+            if (list.start > 1) {
+                list.start = list.start === lastNumber ? lastNumber + 1 : list.start;
+            } else {
+                list.start = lastNumber + 1;
```
Comment (JiuqingSong) [inline packages/roosterjs-editor-dom/lib/list/VListChain.ts:118]: I tried with the latest code but the problem still exists

File: package.json (hunk 1 above)
Comment (juliaroldi) [reply package.json:82]: No it is not. I'll remove it

**Review by JiuqingSong — APPROVED** [review]
